static/index.js

```diff
@@ -27,15 +27,17 @@ async function getSituation() {
 // Create checkbox options for each topic the server supports.
 getTopics().then(topics => {
     const topic_section = document.getElementById("topic_list")
-    Object.entries(topics).forEach(([index, name]) => {
+    var count = 0;
+    topics.forEach(([index, name]) => {
```

Comment:
No, do this properly with real objects. It makes sense in the moment, but you won't remember how this is structured in 6 months.